src/translations/strings.yaml

```diff
@@ -928,6 +928,25 @@ devices:
     value: My devices
     comment: Title for the menu bar when viewing the devices or device limit pages
 
+  excludedAppsTitle:
+    value: Excluded apps
+    comment: Title label for the excluded apps help sheet
+  excludedAppsHeader:
+    value: What are excluded apps? 
+    comment: Header label for the excluded apps help sheet
+  excludedAppsBody1:
+    value: Excluded apps let you turn off VPN protection for specific apps, without turning off your device’s VPN protection.
```

Comment:
Is it just me, or this can be confusing? It makes it sound like the device has a VPN protection that is unrelated to Mozilla VPN, unlike something like this:

> … without turning off VPN protection for your entire device.

Reply:
@traceyvantyghem

Reply:
Sure, we can make this change. Let's update to Flod's suggestion:

Excluded apps let you turn off VPN protection for specific apps, without turning off VPN protection for your entire device.